src/application/health_checks/health_checks.h

```diff
@@ -0,0 +1,28 @@
+#ifndef HEALTH_CHECKS_H
+#define HEALTH_CHECKS_H
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#define ADC1_MAX_COUNTS 65535.0f
+#define ADC1_VREF 3.3f
+#define R_SENSE 0.033f
+#define INA180A3_GAIN 100.0f
+
+static inline float adc1_voltage_v(uint32_t counts) {
+    return (((float)(counts)) / ADC1_MAX_COUNTS) * ADC1_VREF;
+}
+
+static inline float adc1_curr_ma(float voltage) {
+    return (voltage / INA180A3_GAIN / R_SENSE);
```

Comment:
use more constraining parentheses for math instead of assuming order of ops. (i think this is a rule in the BARR standard too)

Reply:
does this still apply when all of the operations are the same? in this case, it's all division